Tests/test_file_tiff_metadata.py

```diff
@@ -155,7 +155,7 @@ def test_iccprofile(self):
 
         im.save(out)
         reloaded = Image.open(out)
-        self.assert_(type(im.info['icc_profile']) is not type(tuple))
+        self.assertTrue(type(im.info['icc_profile']) is not type(tuple))
```

Comment:
I've changed this, and the test now fails. Investigating, I find that you wrote the original test. Any thoughts?